### PR TITLE
Add disclaimer to `remark` feature

### LINKED ISSUE #109: [PE-D] Remark only takes last r/
Corner case of having r/ in remark string causes only the second r/ to be saved

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: chaiwanlin/ped#8

### PR DESCRIPTION
**Pull request by stanley-1**
- Port over @EssWhyy's additions to the UG due to merge conflicts between his PR #143 and the updated UG.
- Resolves #109 by adding disclaimer to `remark` feature (i.e.: only last specified remark will be stored).
- Replaced HelpBox.png with an updated screenshot.
- Replaced all references to `Address Book` with `SociaLite` and other general fixes.

For convenient reading, use this [link](https://stanley-1.github.io/tp/UserGuide.html) to review the UG.

@EssWhyy do have a look and check that your modifications to the UG have been appropriately reflected here! Thereafter, you can merge this version into your PR/branch.

### CODE REVIEW

File: docs/UserGuide.md
Comment (david-eom) [inline docs/UserGuide.md:154]: Thanks for getting rid of the ab references!

**Review by david-eom — APPROVED** [review]
Hi Stanley, thanks for adding the sections on remarks, as well as fixing grammatical mistakes and getting rid of the AB references in the UG! All LGTM, but pending @EssWhyy approval on the accuracy of the description for the find tag section!

**Review by EssWhyy — APPROVED** [review]
Nice, will merge this to #143

**Pull request merged**
